src/jwplayer/players/DesktopArticleVideoPlayer/Attribution.module.css

```diff
@@ -4,6 +4,7 @@
 	display: flex;
 	justify-content: center;
 	padding: 6px;
+	cursor: pointer;
```

Comment:
we should not add `cursor: pointer` wrapper because it's not clickable

Reply:
I added it here to avoid style duplication

Reply:
I can change that. No prob